app/scripts/modules/core/src/config/settings.ts

```diff
@@ -108,6 +108,7 @@ export interface ISpinnakerSettings {
   gateUrl: string;
   gitSources: string[];
   managedDelivery?: {
+    gettingStarted?: string;
```

Comment:
I actually think you named this better calling it `gettingStartedLink` in your component code — could we rename it to that in settings as well? I'd also go for `gettingStartedUrl`.

Reply:
I guess when I have trouble naming things, I tend to throw a bunch of different ones out there just to see what sticks. 👶